### ISSUE TITLE
Crash on change video type on the other side

### ISSUE THREAD

**Issue by nataliia-zarudna** [opentok/opentok-android-sdk-samples#242]
TC:
1. Start a call between A and B, wait till subscribers are connected
2. Call ```publisher.setPublisherVideoType(PublisherKit.PublisherKitVideoType.PublisherKitVideoTypeCamera)``` on side A

AR. Side B has a crash
```
java.lang.IllegalArgumentException: unknown type 0
        at com.opentok.android.Stream$StreamVideoType.fromType(Stream.java:52)
        at com.opentok.android.Session$7.onStreamVideoTypeChanged(Session.java:1420)
        at com.opentok.android.v3.Session$11.run(Session.java:682)
        at android.os.Handler.handleCallback(Handler.java:873)
        at android.os.Handler.dispatchMessage(Handler.java:99)
        at android.os.Looper.loop(Looper.java:193)
        at android.app.ActivityThread.main(ActivityThread.java:6718)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)
```

**Note**: ```PublisherKit.PublisherKitVideoType.PublisherKitVideoTypeScreen``` value does not provoke a crash.

In decompiled code I've noticed that in Session.StreamPropertiesListener#onStreamVideoTypeChanged there is a call of StreamVideoType.fromType(**videoType.ordinal()**), but in StreamVideoType#fromType type is searched by **videoType.getVideoType()**. As camera type has ordinal=0/type=1 and screen has ordinal=1/type=2 when type is set to PublisherKit.PublisherKitVideoType.PublisherKitVideoTypeScreen on the other side type is resolved as StreamVideoType.StreamVideoTypeCamera. On the other hand when type is set to PublisherKit.PublisherKitVideoType.PublisherKitVideoTypeCamera on the other side it provokes crash as there is no value with video type 0.

Version: 2.16.1.

**Comment by nataliia-zarudna** [comment]
**Sample**: https://github.com/opentok/opentok-android-sdk-samples/tree/master/Screen-Sharing

**Steps**:
1. Start a call between A and B, wait till subscribers are connected
2. Call publisher.setPublisherVideoType(PublisherKit.PublisherKitVideoType.PublisherKitVideoTypeCamera) on side A

**Crash was reproduced on**
1. Motorola 1st Gen Android 5.0
2. Pixel 2 Android 9.0 (edited)

**Comment by robjperez** [comment]
This should be fixed on 2.17

Please reopen if it still crashes
